static/js/portico/team.js

```diff
@@ -58,6 +58,10 @@ function get_display_name(contributor) {
     return contributor.name;
 }
 
+function exclude_github_username(contributor, excluded_username) {
+    return contributor.github_username !== excluded_username;
+}
```

Comment:
Let's have the function be `exclude_bot_contributors` and have it exclude dependabot -- I think that's a more extensible design if we end up with other automation creating commits in the future.